File: Coding/coding modul 6/PRAK602-2410817110007-MUHAMMADHAMKAAKBARI.c
#include <stdio.h>

int main() {
    int n; // Jumlah ruangan
    scanf("%d", &n);

    int zetsu[n]; // Array untuk menyimpan jumlah awal Zetsu putih
    for (int i = 0; i < n; i++) {
        scanf("%d", &zetsu[i]);
    }

    // Menghitung jumlah Zetsu putih setelah membelah diri
    for (int i = 0; i < n; i++) {
        zetsu[i] *= (i + 1); // Perkalian dengan label ruangan (index + 1)
    }

    printf("\n");

    // Menampilkan output
    for (int i = 0; i < n; i++) {
        printf("%d ", zetsu[i]);
    }
    printf("\n");

    return 0;
}

File: Coding/coding modul 6/PRAK602-2410817110007-MUHAMMADHAMKAAKBARI.py
# Input jumlah ruangan
n = int(input())

# Input jumlah awal Zetsu putih di setiap ruangan
zetsu = list(map(int, input().split()))

# Menghitung jumlah Zetsu putih setelah membelah diri
for i in range(n):
    zetsu[i] *= (i + 1)  # Perkalian dengan label ruangan (index + 1)

print("\n")

# Menampilkan output
print(" ".join(map(str, zetsu)))
